fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -681,7 +682,9 @@ class RYWImpl {
 				break;
 
 			if (it.is_unknown_range()) {
-				if (limits.hasByteLimit() && result.size() && itemsPastEnd >= 1 - end.offset) {
+				if (limits.hasByteLimit() &&
+				    (!origLimits.minRows || (result.size() && origLimits.minRows <= result.size())) &&
```

Comment:
I'm actually confused that this fixes the bug we talked about, but I'm not 100% sure what the exact api contract is here.
I'd have to think about it but I feel like there could be some cases where the original behavior is desired instead of this?
Like if i am using min rows to say I want 1000 rows but i'm only doing chunks of ~100-rows per read in the same transaction?